tests/CMakeLists.txt

```diff
@@ -6,31 +6,55 @@ function(add_vue_test TestName)
   )
 endfunction()
 
-function(add_cypress_test TestName)
+function(cypress_shared)
```

Comment:
I know that we shouldn't be calling this more than once, and that it's worrying that this value changes during the runtime of the cmake configure step. Be it still might be better to rename this function to `set_app_url()` or something that makes it more clear as of what it does.

Reply:
Good idea.